recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
+  version: 2.0.12
+
+source:
+  fn: "uwsgi-2.0.12.tar.gz"
+  url: "https://pypi.python.org/packages/0c/1c/44849e293e367a157f1ad863cee02b4b865840543254d8fae3ecdebdbdb9/uwsgi-2.0.12.tar.gz"
+  md5: 1451cab954bad0d7d7429e4d2c84b5df
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - python
+    # - jansson
+    - libxml2
+    - openssl 1.0.2*
+    - pcre
+    - yaml
+    - zlib
+  run:
+    - python
+    # - jansson
+    - libxml2
+    - openssl 1.0.2*
+    - pcre
+    - yaml
+    - zlib
+
+test:
+  commands:
+    - uwsgi --version
+
+about:
+  home: https://github.com/unbit/uwsgi
```

Comment:
We only recently started discussing this ( https://github.com/conda-forge/conda-forge.github.io/issues/119 ). The current view is we expect to see `home`, `license`, and `summary`. We have no plans on forcing people to use the other keys, but we won't stop them. Some of them, like these URL keys, are often easily found by visiting `home` if they aren't the same (e.g. a GitHub repo URL). In other cases, these may not even exist. Though we have been giving some serious thought to requiring `license_file` as so many license have some clause about distributing the license even with binaries. So, that one would be worth adding. The rest I leave to your own discretion.